docs/getting-started/contributors.md

```diff
@@ -35,6 +35,10 @@ $ yarn install  # fetch dependency packages - may take a while
 $ yarn tsc      # does a first run of type generation and checks
 ```
 
+If yarn install does not complete, you might need to download some of the dependencies.
+The best way to find out which dependencies are missing is by looking through the logs
+of the yarn install, and finding which dependency is missing.
```

Comment:
Uh, this isn't really true is it? 😅 Could you give an example of one of those dependencies? If we're talking OS stuff like `python` perhaps we could word in a bit more differently and specifically? Making a list of any dependencies that might be missing.

Reply:
How would I be able to bypass the spell checker on these words @Rugvip?